osf/models/registrations.py

```diff
@@ -632,6 +633,7 @@ def _write_registration_action(self, from_state, to_state, initiated_by, comment
             comment=comment
         )
         action.save()
+        RegistriesModerationMetrics.record_transitions(action)
```

Comment:
Should we only be writing these when there's an actual Moderator action? i.e. omit `SUBMIT` and `REQUEST_WITHDRAWAL` triggers?

The only reason I can think to write metrics for submissions is if we later decide we want to compute time-to-respond. Everything else can be inferred from the moderator triggers, and we'd be cutting our storage roughly in half.

Reply:
We don't really know what this data is going to be used for, so it's better to be comprehensive. I don't think storage is going to be an issue, it's pretty fractional amount compared to the other metrics.